CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 - PR #222 - Improved performance for various window functions
 - PR #235 - Improve Wavelets functions performance
 - PR #242 - Add PyTorch disclaimer to notebook
+- PR #243 - Improve Peak Finding function performance
+- PR #242 - Add PyTorch disclaimer to notebook
```

Comment:
This is a super nit, but can you switch the order of these PR lists? I.e. put 242 before 243?

Reply:
Yup!